validator/errorbundler.py

```diff
@@ -74,6 +72,8 @@ def __init__(self, determined=True, listed=True, instant=False,
 
         self.supported_versions = self.for_appversions = for_appversions
 
+    SEVERITIES = {"trivial", "low", "medium", "high"}
```

Comment:
I haven't decided if I'm going to keep that yet. I added it for the moment because there was a warning that was signing-related, but I didn't want to trigger manual review. If it stays, the idea would be that it would show up as a signing-related message, but wouldn't prevent signing. Or maybe a bunch of them would, but a single one wouldn't.